doc/iterations/2023.10/shapes/color_management_in_view.adoc

```diff
@@ -0,0 +1,106 @@
+= (M) Color Management in View
+
+== Problems
+
+It is possible to set `None` for color properties of a Node/Edge Description.
+Depending on the case, `None` means `black`, `white` or `transparent`.
+
+It is possible to set `None` for color properties of a Form Widget Description.
+Depending on the case, `None` means `theme.primary`, `theme.secondary` or `transparent`.
+
+Some color properties of a Node/Edge/Widget Description are mandatory, which means when `None` is selected, an error message is displayed.
+That's ok but we should review which color property is really mandatory or not.
+
+Furthermore, when a View is created non Color Palette is provided by default.
+The result for color properties of a Node/Edge/Widget Description is that there is no other options than `None`. 
+
+== Key results
+
+* Specifier always know which color will be displayed at the end, corresponding exactly to what is set for each color property.
+* A set of predefined colors is accessible in color properties.
+
+== Solutions
+
+1) Preferred solution
+
+`None` doesn't exist anymore as a possible value for color properties.
+
+All color properties are mandatory.
```

Comment:
I would go for this solution over any other for one simple fact: It stops us from setting magic values everywhere. The specifier has to give us a proper value.

Reply:
+1